server/src/immich/controllers/app.controller.ts

```diff
@@ -1,15 +1,34 @@
 import { SystemConfigService } from '@app/domain';
-import { Controller, HttpCode, HttpStatus, Post } from '@nestjs/common';
+import { Controller, Get, Header, HttpCode, HttpStatus, Post } from '@nestjs/common';
 import { ApiExcludeEndpoint } from '@nestjs/swagger';
+import { PublicRoute } from '../app.guard';
 
 @Controller()
 export class AppController {
-  constructor(private configService: SystemConfigService) {}
+  constructor(private service: SystemConfigService) {}
+
+  @ApiExcludeEndpoint()
+  @Get('.well-known/immich')
+  getImmichWellKnown() {
+    return {
+      api: {
+        endpoint: '/api',
+      },
+    };
+  }
```

Comment:
What is this endpoint? lol.

Reply:
People don't want to type in /api on the mobile app, so this is a "discovery" endpoint to figure out the location of the API.

Reply:
Oh I see that makes sense!